scripts/sync_packages.sh

```diff
@@ -1,35 +1,41 @@
 #!/bin/bash
-robot_name="${HOSTNAME//-b1}"
```

Comment:
the problem with using `robot_name` from `$HOSTNAME` is that we cannot use the sync script on any other pc except b1, e.g. the torso pcs.
Do you have a suggestion how to handle that without using `ROBOT`? can't we set `ROBOT` by default for the root user with our install scripts?